### DISCUSSION TITLE
How can you see what approvals you have for your deepgram project?

### DISCUSSION THREAD

**Discussion by jameshorton2486** [deepgram/community#596] [General help] [answered]
I am trying to utilize the live streaming option on Deepgram.  I have this project that I created several months ago.  e4d1b83d-9bae-4955-8028-1559d7358e2d.  I am having trouble when I send my API key to deepgram for a live streaming script I created in Python.  I am very new to all of this and would really appreciate it if someone told me how I can check the approvals that my project currently have.  Are there other things that I also need to get approval from deepgram for.  I created a new project but my funds did not apply to the new project.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @jameshorton2486, you should not need any additional permissions granted, everything is self-service. If you are seeing any specific error messages, please feel free to share the errors and any associated Deepgram request ID for troubleshooting. It may also help to create a new API key to make sure it is under your expected project, has not expired, etc.
